src/solvers/dg/3d/dg.jl

```diff
@@ -931,6 +939,7 @@ function analyze_solution(dg::Dg3D, mesh::TreeMesh, time::Real, dt::Real, step::
           " Time/DOF/step:  " * @sprintf("%10.8e s", runtime_relative))
   println(" sim. time:      " * @sprintf("%10.8e", time))
 
+  println(" #DOF:           " * @sprintf("% 14d", dg.n_elements*(polydeg(dg)+1)^ndims(mesh)))
```

Comment:
```suggestion
  println(" #DOF:           " * @sprintf("% 14d", ndofs(dg)))
```

From here:
https://github.com/trixi-framework/Trixi.jl/blob/dd7502e58ab880968f59b0e510351e866bdba6ff/src/solvers/dg/dg.jl#L20